solr/solr-ref-guide/modules/indexing-guide/pages/post-tool.adoc

```diff
@@ -161,12 +166,12 @@ Index a PDF as the user "solr" with password "SolrRocks":
 
 [source,bash]
 ----
-bin/post -u solr:SolrRocks -c gettingstarted a.pdf
+bin/solr post -u solr:SolrRocks -url http://localhost:8983/solr/gettingstarted/update a.pdf
 ----
 
 == Post Tool Windows Support
 
-`bin/post` is a Unix shell script and as such cannot be used directly on Windows.
+`bin/solr post` is a Unix shell script and as such cannot be used directly on Windows.
```

Comment:
Not true :) No need for this entire paragraph I guess now..

Reply:
thanks..   THough I guess I better make sure bin/solr.cmd post works!